examples/pump-app/ti/cc13x4_26x4/main/AppTask.cpp

```diff
@@ -223,7 +223,7 @@ int AppTask::Init()
     cc13xx_26xxLogInit();
 
     // Init Chip memory management before the stack
-    Platform::MemoryInit();
+    TEMPORARY_RETURN_IGNORED Platform::MemoryInit();
```

Comment:
I think that memory initialization failure should be fatal, otherwise you might face undefined behavior...

Reply:
Fixed